src/main/java/UITestFramework/CreateSession.java

```diff
@@ -44,7 +47,30 @@
 	Properties configProp = new Properties();
 	String OS;
 
+	public WebDriver getDriver() {
```

Comment:
Please add a single line comment on top of this method -

`/**
	 * Returns the instance of the webdriver.
	 * @return webdriver instance
	 */`

Reply:
Added the comment.